mopidy_mpd/dispatcher.py

```diff
@@ -155,7 +155,7 @@ def _add_ok_filter(self, request, response, filter_chain):
         return response
 
     def _has_error(self, response):
-        return response and response[-1].startswith("ACK")
+        return response and type(response[-1]) == str and response[-1].startswith("ACK")
```

Comment:
I think you’re supposed to use `is` to compare types, not `==`.

Reply:
Done! :) Thanks for the feedback and sorry for the late reply, the week's been busy.